<test-run asserts="0" clr-version="3.6.8 (default, Jan 14 2019, 11:02:34) &#10;[GCC 8.0.1 20180414 (experimental) [trunk revision 259383]]" duration="9" end-time="2019-08-10 15:15:06.880762" engine-version="3.6.2" failed="0" id="2" inconclusive="0" passed="13" result="Passed" skipped="1" start-time="2019-08-10 15:14:57.421202" testcasecount="14" total="14"><command-line>/home/adetomi/Desktop/pytest-nunit/.venv/bin/pytest tests/integration --nunitxml=text.xml</command-line><filter /><test-suite asserts="0" classname="" duration="9.45956" end-time="2019-08-10 15:15:06.880762" failed="0" fullname="pytest" id="3" inconclusive="0" label="" methodname="" name="pytest" passed="13" result="Passed" runstate="Runnable" skipped="1" start-time="2019-08-10 15:14:57.421202" testcasecount="14" total="14" type="Assembly" warnings="0"><properties><property name="python_version" value="3.6.8 (default, Jan 14 2019, 11:02:34) &#10;[GCC 8.0.1 20180414 (experimental) [trunk revision 259383]]" /></properties><test-case asserts="0" classname="tests/integration/test_azure.py" duration="0.001055" end-time="2019-08-10 15:14:58.031843" fullname="tests/integration/test_azure.py::test_attachment" id="100" label="" methodname="test_attachment" name="tests/integration/test_azure.py::test_attachment" result="Passed" runstate="Runnable" seed="1" start-time="2019-08-10 15:14:58.030788"><properties><property name="python-version" value="3.6.8 (default, Jan 14 2019, 11:02:34) &#10;[GCC 8.0.1 20180414 (experimental) [trunk revision 259383]]" /></properties><environment clr-version="3.6.8 (default, Jan 14 2019, 11:02:34) &#10;[GCC 8.0.1 20180414 (experimental) [trunk revision 259383]]" culture="en_US" cwd="/home/adetomi/Desktop/pytest-nunit" framework-version="3.6.2" machine-name="x86_64" os-architecture="64bit" os-version="4.15.0-20-generic" platform="Linux" uiculture="en_US" user="" user-domain="" /><failure><message><![CDATA[]]></message><stack-trace><![CDATA[None]]></stack-trace></failure><reason><message><![CDATA[]]></message></reason><output><![CDATA[]]></output><attachments><attachment><filePath>/home/adetomi/Desktop/pytest-nunit/tests/integration/fixture.gif</filePath><description>PBJT</description></attachment></attachments></test-case><test-case asserts="0" classname="tests/integration/test_azure.py" duration="0.001038" end-time="2019-08-10 15:14:58.033754" fullname="tests/integration/test_azure.py::test_property" id="101" label="" methodname="test_property" name="tests/integration/test_azure.py::test_property" result="Passed" runstate="Runnable" seed="1" start-time="2019-08-10 15:14:58.032716"><properties><property name="python-version" value="3.6.8 (default, Jan 14 2019, 11:02:34) &#10;[GCC 8.0.1 20180414 (experimental) [trunk revision 259383]]" /><property name="test" value="value" /></properties><environment clr-version="3.6.8 (default, Jan 14 2019, 11:02:34) &#10;[GCC 8.0.1 20180414 (experimental) [trunk revision 259383]]" culture="en_US" cwd="/home/adetomi/Desktop/pytest-nunit" framework-version="3.6.2" machine-name="x86_64" os-architecture="64bit" os-version="4.15.0-20-generic" platform="Linux" uiculture="en_US" user="" user-domain="" /><failure><message><![CDATA[]]></message><stack-trace><![CDATA[None]]></stack-trace></failure><reason><message><![CDATA[]]></message></reason><output><![CDATA[]]></output></test-case><test-case asserts="0" classname="tests/integration/test_azure.py" duration="0.02958" end-time="2019-08-10 15:14:58.064707" fullname="tests/integration/test_azure.py::test_failure" id="102" label="" methodname="test_failure" name="tests/integration/test_azure.py::test_failure" result="Passed" runstate="Runnable" seed="1" start-time="2019-08-10 15:14:58.035127"><properties><property name="python-version" value="3.6.8 (default, Jan 14 2019, 11:02:34) &#10;[GCC 8.0.1 20180414 (experimental) [trunk revision 259383]]" /></properties><environment clr-version="3.6.8 (default, Jan 14 2019, 11:02:34) &#10;[GCC 8.0.1 20180414 (experimental) [trunk revision 259383]]" culture="en_US" cwd="/home/adetomi/Desktop/pytest-nunit" framework-version="3.6.2" machine-name="x86_64" os-architecture="64bit" os-version="4.15.0-20-generic" platform="Linux" uiculture="en_US" user="" user-domain="" /><failure><message><![CDATA[@pytest.mark.xfail
    def test_failure():
>       assert 1 == 0
E       assert 1 == 0

tests/integration/test_azure.py:28: AssertionError]]></message><stack-trace><![CDATA[/home/adetomi/Desktop/pytest-nunit/tests/integration/test_azure.py:28: assert 1 == 0]]></stack-trace></failure><reason><message><![CDATA[]]></message></reason><output><![CDATA[]]></output></test-case><test-case asserts="0" classname="tests/integration/test_azure.py" duration="0" end-time="2019-08-10 15:14:58.066209" fullname="tests/integration/test_azure.py::test_skip" id="103" label="" methodname="test_skip" name="tests/integration/test_azure.py::test_skip" result="Skipped" runstate="Skipped" seed="1" start-time="2019-08-10 15:14:58.065648"><properties><property name="python-version" value="3.6.8 (default, Jan 14 2019, 11:02:34) &#10;[GCC 8.0.1 20180414 (experimental) [trunk revision 259383]]" /></properties><environment clr-version="3.6.8 (default, Jan 14 2019, 11:02:34) &#10;[GCC 8.0.1 20180414 (experimental) [trunk revision 259383]]" culture="en_US" cwd="/home/adetomi/Desktop/pytest-nunit" framework-version="3.6.2" machine-name="x86_64" os-architecture="64bit" os-version="4.15.0-20-generic" platform="Linux" uiculture="en_US" user="" user-domain="" /><failure><message><![CDATA[Skipped: Example skip]]></message><stack-trace><![CDATA[tests/integration/test_azure.py::30]]></stack-trace></failure><reason><message><![CDATA[]]></message></reason><output><![CDATA[]]></output></test-case><test-case asserts="0" classname="tests/integration/test_config.py" duration="3.26865" end-time="2019-08-10 15:15:01.344507" fullname="tests/integration/test_config.py::test_prefix" id="104" label="" methodname="test_prefix" name="tests/integration/test_config.py::test_prefix" result="Passed" runstate="Runnable" seed="1" start-time="2019-08-10 15:14:58.075857"><properties><property name="python-version" value="3.6.8 (default, Jan 14 2019, 11:02:34) &#10;[GCC 8.0.1 20180414 (experimental) [trunk revision 259383]]" /></properties><environment clr-version="3.6.8 (default, Jan 14 2019, 11:02:34) &#10;[GCC 8.0.1 20180414 (experimental) [trunk revision 259383]]" culture="en_US" cwd="/home/adetomi/Desktop/pytest-nunit" framework-version="3.6.2" machine-name="x86_64" os-architecture="64bit" os-version="4.15.0-20-generic" platform="Linux" uiculture="en_US" user="" user-domain="" /><failure><message><![CDATA[]]></message><stack-trace><![CDATA[None]]></stack-trace></failure><reason><message><![CDATA[[35mDEBUG   [0m __name__:plugin.py:86 record_test_report:<TestReport 'test_prefix.py::test_basic' when='setup' outcome='passed'>
[35mDEBUG   [0m __name__:plugin.py:86 record_test_report:<TestReport 'test_prefix.py::test_basic' when='call' outcome='passed'>
[35mDEBUG   [0m __name__:plugin.py:86 record_test_report:<TestReport 'test_prefix.py::test_basic' when='teardown' outcome='passed'>]]></message></reason><output><![CDATA[[35mDEBUG   [0m __name__:plugin.py:86 record_test_report:<TestReport 'test_prefix.py::test_basic' when='setup' outcome='passed'>
[35mDEBUG   [0m __name__:plugin.py:86 record_test_report:<TestReport 'test_prefix.py::test_basic' when='call' outcome='passed'>
[35mDEBUG   [0m __name__:plugin.py:86 record_test_report:<TestReport 'test_prefix.py::test_basic' when='teardown' outcome='passed'>]]></output></test-case><test-case asserts="0" classname="tests/integration/test_properties.py" duration="0.305647" end-time="2019-08-10 15:15:01.652607" fullname="tests/integration/test_properties.py::test_basic_property" id="105" label="" methodname="test_basic_property" name="tests/integration/test_properties.py::test_basic_property" result="Passed" runstate="Runnable" seed="1" start-time="2019-08-10 15:15:01.346960"><properties><property name="python-version" value="3.6.8 (default, Jan 14 2019, 11:02:34) &#10;[GCC 8.0.1 20180414 (experimental) [trunk revision 259383]]" /></properties><environment clr-version="3.6.8 (default, Jan 14 2019, 11:02:34) &#10;[GCC 8.0.1 20180414 (experimental) [trunk revision 259383]]" culture="en_US" cwd="/home/adetomi/Desktop/pytest-nunit" framework-version="3.6.2" machine-name="x86_64" os-architecture="64bit" os-version="4.15.0-20-generic" platform="Linux" uiculture="en_US" user="" user-domain="" /><failure><message><![CDATA[]]></message><stack-trace><![CDATA[None]]></stack-trace></failure><reason><message><![CDATA[[35mDEBUG   [0m __name__:plugin.py:86 record_test_report:<TestReport 'test_basic_property.py::test_basic' when='setup' outcome='passed'>
[35mDEBUG   [0m __name__:plugin.py:86 record_test_report:<TestReport 'test_basic_property.py::test_basic' when='call' outcome='passed'>
[35mDEBUG   [0m __name__:plugin.py:86 record_test_report:<TestReport 'test_basic_property.py::test_basic' when='teardown' outcome='passed'>]]></message></reason><output><![CDATA[[35mDEBUG   [0m __name__:plugin.py:86 record_test_report:<TestReport 'test_basic_property.py::test_basic' when='setup' outcome='passed'>
[35mDEBUG   [0m __name__:plugin.py:86 record_test_report:<TestReport 'test_basic_property.py::test_basic' when='call' outcome='passed'>
[35mDEBUG   [0m __name__:plugin.py:86 record_test_report:<TestReport 'test_basic_property.py::test_basic' when='teardown' outcome='passed'>]]></output></test-case><test-case asserts="0" classname="tests/integration/test_properties.py" duration="0.250158" end-time="2019-08-10 15:15:01.904818" fullname="tests/integration/test_properties.py::test_attachment" id="106" label="" methodname="test_attachment" name="tests/integration/test_properties.py::test_attachment" result="Passed" runstate="Runnable" seed="1" start-time="2019-08-10 15:15:01.654660"><properties><property name="python-version" value="3.6.8 (default, Jan 14 2019, 11:02:34) &#10;[GCC 8.0.1 20180414 (experimental) [trunk revision 259383]]" /></properties><environment clr-version="3.6.8 (default, Jan 14 2019, 11:02:34) &#10;[GCC 8.0.1 20180414 (experimental) [trunk revision 259383]]" culture="en_US" cwd="/home/adetomi/Desktop/pytest-nunit" framework-version="3.6.2" machine-name="x86_64" os-architecture="64bit" os-version="4.15.0-20-generic" platform="Linux" uiculture="en_US" user="" user-domain="" /><failure><message><![CDATA[]]></message><stack-trace><![CDATA[None]]></stack-trace></failure><reason><message><![CDATA[[35mDEBUG   [0m __name__:plugin.py:86 record_test_report:<TestReport 'test_attachment.py::test_basic' when='setup' outcome='passed'>
[35mDEBUG   [0m __name__:plugin.py:86 record_test_report:<TestReport 'test_attachment.py::test_basic' when='call' outcome='passed'>
[35mDEBUG   [0m __name__:plugin.py:86 record_test_report:<TestReport 'test_attachment.py::test_basic' when='teardown' outcome='passed'>]]></message></reason><output><![CDATA[[35mDEBUG   [0m __name__:plugin.py:86 record_test_report:<TestReport 'test_attachment.py::test_basic' when='setup' outcome='passed'>
[35mDEBUG   [0m __name__:plugin.py:86 record_test_report:<TestReport 'test_attachment.py::test_basic' when='call' outcome='passed'>
[35mDEBUG   [0m __name__:plugin.py:86 record_test_report:<TestReport 'test_attachment.py::test_basic' when='teardown' outcome='passed'>]]></output></test-case><test-case asserts="0" classname="tests/integration/test_properties.py" duration="3.263744" end-time="2019-08-10 15:15:05.170667" fullname="tests/integration/test_properties.py::test_slow_test" id="107" label="" methodname="test_slow_test" name="tests/integration/test_properties.py::test_slow_test" result="Passed" runstate="Runnable" seed="1" start-time="2019-08-10 15:15:01.906923"><properties><property name="python-version" value="3.6.8 (default, Jan 14 2019, 11:02:34) &#10;[GCC 8.0.1 20180414 (experimental) [trunk revision 259383]]" /></properties><environment clr-version="3.6.8 (default, Jan 14 2019, 11:02:34) &#10;[GCC 8.0.1 20180414 (experimental) [trunk revision 259383]]" culture="en_US" cwd="/home/adetomi/Desktop/pytest-nunit" framework-version="3.6.2" machine-name="x86_64" os-architecture="64bit" os-version="4.15.0-20-generic" platform="Linux" uiculture="en_US" user="" user-domain="" /><failure><message><![CDATA[]]></message><stack-trace><![CDATA[None]]></stack-trace></failure><reason><message><![CDATA[[35mDEBUG   [0m __name__:plugin.py:86 record_test_report:<TestReport 'test_slow_test.py::test_basic' when='setup' outcome='passed'>
[35mDEBUG   [0m __name__:plugin.py:86 record_test_report:<TestReport 'test_slow_test.py::test_basic' when='call' outcome='passed'>
[35mDEBUG   [0m __name__:plugin.py:86 record_test_report:<TestReport 'test_slow_test.py::test_basic' when='teardown' outcome='passed'>]]></message></reason><output><![CDATA[[35mDEBUG   [0m __name__:plugin.py:86 record_test_report:<TestReport 'test_slow_test.py::test_basic' when='setup' outcome='passed'>
[35mDEBUG   [0m __name__:plugin.py:86 record_test_report:<TestReport 'test_slow_test.py::test_basic' when='call' outcome='passed'>
[35mDEBUG   [0m __name__:plugin.py:86 record_test_report:<TestReport 'test_slow_test.py::test_basic' when='teardown' outcome='passed'>]]></output></test-case><test-case asserts="0" classname="tests/integration/test_schema.py" duration="0.345452" end-time="2019-08-10 15:15:05.518564" fullname="tests/integration/test_schema.py::test_basic_against_reference_schema" id="108" label="" methodname="test_basic_against_reference_schema" name="tests/integration/test_schema.py::test_basic_against_reference_schema" result="Passed" runstate="Runnable" seed="1" start-time="2019-08-10 15:15:05.173112"><properties><property name="python-version" value="3.6.8 (default, Jan 14 2019, 11:02:34) &#10;[GCC 8.0.1 20180414 (experimental) [trunk revision 259383]]" /></properties><environment clr-version="3.6.8 (default, Jan 14 2019, 11:02:34) &#10;[GCC 8.0.1 20180414 (experimental) [trunk revision 259383]]" culture="en_US" cwd="/home/adetomi/Desktop/pytest-nunit" framework-version="3.6.2" machine-name="x86_64" os-architecture="64bit" os-version="4.15.0-20-generic" platform="Linux" uiculture="en_US" user="" user-domain="" /><failure><message><![CDATA[]]></message><stack-trace><![CDATA[None]]></stack-trace></failure><reason><message><![CDATA[[35mDEBUG   [0m __name__:plugin.py:86 record_test_report:<TestReport 'test_basic_against_reference_schema.py::test_basic' when='setup' outcome='passed'>
[35mDEBUG   [0m __name__:plugin.py:86 record_test_report:<TestReport 'test_basic_against_reference_schema.py::test_basic' when='call' outcome='passed'>
[35mDEBUG   [0m __name__:plugin.py:86 record_test_report:<TestReport 'test_basic_against_reference_schema.py::test_basic' when='teardown' outcome='passed'>]]></message></reason><output><![CDATA[[35mDEBUG   [0m __name__:plugin.py:86 record_test_report:<TestReport 'test_basic_against_reference_schema.py::test_basic' when='setup' outcome='passed'>
[35mDEBUG   [0m __name__:plugin.py:86 record_test_report:<TestReport 'test_basic_against_reference_schema.py::test_basic' when='call' outcome='passed'>
[35mDEBUG   [0m __name__:plugin.py:86 record_test_report:<TestReport 'test_basic_against_reference_schema.py::test_basic' when='teardown' outcome='passed'>]]></output></test-case><test-case asserts="0" classname="tests/integration/test_tests.py" duration="0.274258" end-time="2019-08-10 15:15:05.795246" fullname="tests/integration/test_tests.py::test_passing_test" id="109" label="" methodname="test_passing_test" name="tests/integration/test_tests.py::test_passing_test" result="Passed" runstate="Runnable" seed="1" start-time="2019-08-10 15:15:05.520988"><properties><property name="python-version" value="3.6.8 (default, Jan 14 2019, 11:02:34) &#10;[GCC 8.0.1 20180414 (experimental) [trunk revision 259383]]" /></properties><environment clr-version="3.6.8 (default, Jan 14 2019, 11:02:34) &#10;[GCC 8.0.1 20180414 (experimental) [trunk revision 259383]]" culture="en_US" cwd="/home/adetomi/Desktop/pytest-nunit" framework-version="3.6.2" machine-name="x86_64" os-architecture="64bit" os-version="4.15.0-20-generic" platform="Linux" uiculture="en_US" user="" user-domain="" /><failure><message><![CDATA[]]></message><stack-trace><![CDATA[None]]></stack-trace></failure><reason><message><![CDATA[[35mDEBUG   [0m __name__:plugin.py:86 record_test_report:<TestReport 'test_passing_test.py::test_pass' when='setup' outcome='passed'>
[35mDEBUG   [0m __name__:plugin.py:86 record_test_report:<TestReport 'test_passing_test.py::test_pass' when='call' outcome='passed'>
[35mDEBUG   [0m __name__:plugin.py:86 record_test_report:<TestReport 'test_passing_test.py::test_pass' when='teardown' outcome='passed'>]]></message></reason><output><![CDATA[[35mDEBUG   [0m __name__:plugin.py:86 record_test_report:<TestReport 'test_passing_test.py::test_pass' when='setup' outcome='passed'>
[35mDEBUG   [0m __name__:plugin.py:86 record_test_report:<TestReport 'test_passing_test.py::test_pass' when='call' outcome='passed'>
[35mDEBUG   [0m __name__:plugin.py:86 record_test_report:<TestReport 'test_passing_test.py::test_pass' when='teardown' outcome='passed'>]]></output></test-case><test-case asserts="0" classname="tests/integration/test_tests.py" duration="0.279579" end-time="2019-08-10 15:15:06.077039" fullname="tests/integration/test_tests.py::test_failing_test" id="110" label="" methodname="test_failing_test" name="tests/integration/test_tests.py::test_failing_test" result="Passed" runstate="Runnable" seed="1" start-time="2019-08-10 15:15:05.797460"><properties><property name="python-version" value="3.6.8 (default, Jan 14 2019, 11:02:34) &#10;[GCC 8.0.1 20180414 (experimental) [trunk revision 259383]]" /></properties><environment clr-version="3.6.8 (default, Jan 14 2019, 11:02:34) &#10;[GCC 8.0.1 20180414 (experimental) [trunk revision 259383]]" culture="en_US" cwd="/home/adetomi/Desktop/pytest-nunit" framework-version="3.6.2" machine-name="x86_64" os-architecture="64bit" os-version="4.15.0-20-generic" platform="Linux" uiculture="en_US" user="" user-domain="" /><failure><message><![CDATA[]]></message><stack-trace><![CDATA[None]]></stack-trace></failure><reason><message><![CDATA[[35mDEBUG   [0m __name__:plugin.py:86 record_test_report:<TestReport 'test_failing_test.py::test_fail' when='setup' outcome='passed'>
[35mDEBUG   [0m __name__:plugin.py:86 record_test_report:<TestReport 'test_failing_test.py::test_fail' when='call' outcome='failed'>
[35mDEBUG   [0m __name__:plugin.py:86 record_test_report:<TestReport 'test_failing_test.py::test_fail' when='teardown' outcome='passed'>]]></message></reason><output><![CDATA[[35mDEBUG   [0m __name__:plugin.py:86 record_test_report:<TestReport 'test_failing_test.py::test_fail' when='setup' outcome='passed'>
[35mDEBUG   [0m __name__:plugin.py:86 record_test_report:<TestReport 'test_failing_test.py::test_fail' when='call' outcome='failed'>
[35mDEBUG   [0m __name__:plugin.py:86 record_test_report:<TestReport 'test_failing_test.py::test_fail' when='teardown' outcome='passed'>]]></output></test-case><test-case asserts="0" classname="tests/integration/test_tests.py" duration="0.262297" end-time="2019-08-10 15:15:06.341605" fullname="tests/integration/test_tests.py::test_skipped_test" id="111" label="" methodname="test_skipped_test" name="tests/integration/test_tests.py::test_skipped_test" result="Passed" runstate="Runnable" seed="1" start-time="2019-08-10 15:15:06.079308"><properties><property name="python-version" value="3.6.8 (default, Jan 14 2019, 11:02:34) &#10;[GCC 8.0.1 20180414 (experimental) [trunk revision 259383]]" /></properties><environment clr-version="3.6.8 (default, Jan 14 2019, 11:02:34) &#10;[GCC 8.0.1 20180414 (experimental) [trunk revision 259383]]" culture="en_US" cwd="/home/adetomi/Desktop/pytest-nunit" framework-version="3.6.2" machine-name="x86_64" os-architecture="64bit" os-version="4.15.0-20-generic" platform="Linux" uiculture="en_US" user="" user-domain="" /><failure><message><![CDATA[]]></message><stack-trace><![CDATA[None]]></stack-trace></failure><reason><message><![CDATA[[35mDEBUG   [0m __name__:plugin.py:86 record_test_report:<TestReport 'test_skipped_test.py::test_skip' when='setup' outcome='skipped'>
[35mDEBUG   [0m __name__:plugin.py:103 skipping : ('test_skipped_test.py', 2, 'Skipped: unconditional skip')
[35mDEBUG   [0m __name__:plugin.py:86 record_test_report:<TestReport 'test_skipped_test.py::test_skip' when='teardown' outcome='passed'>]]></message></reason><output><![CDATA[[35mDEBUG   [0m __name__:plugin.py:86 record_test_report:<TestReport 'test_skipped_test.py::test_skip' when='setup' outcome='skipped'>
[35mDEBUG   [0m __name__:plugin.py:103 skipping : ('test_skipped_test.py', 2, 'Skipped: unconditional skip')
[35mDEBUG   [0m __name__:plugin.py:86 record_test_report:<TestReport 'test_skipped_test.py::test_skip' when='teardown' outcome='passed'>]]></output></test-case><test-case asserts="0" classname="tests/integration/test_tests.py" duration="0.273875" end-time="2019-08-10 15:15:06.617806" fullname="tests/integration/test_tests.py::test_all_outcomes" id="112" label="" methodname="test_all_outcomes" name="tests/integration/test_tests.py::test_all_outcomes" result="Passed" runstate="Runnable" seed="1" start-time="2019-08-10 15:15:06.343931"><properties><property name="python-version" value="3.6.8 (default, Jan 14 2019, 11:02:34) &#10;[GCC 8.0.1 20180414 (experimental) [trunk revision 259383]]" /></properties><environment clr-version="3.6.8 (default, Jan 14 2019, 11:02:34) &#10;[GCC 8.0.1 20180414 (experimental) [trunk revision 259383]]" culture="en_US" cwd="/home/adetomi/Desktop/pytest-nunit" framework-version="3.6.2" machine-name="x86_64" os-architecture="64bit" os-version="4.15.0-20-generic" platform="Linux" uiculture="en_US" user="" user-domain="" /><failure><message><![CDATA[]]></message><stack-trace><![CDATA[None]]></stack-trace></failure><reason><message><![CDATA[[35mDEBUG   [0m __name__:plugin.py:86 record_test_report:<TestReport 'test_all_outcomes.py::test_pass' when='setup' outcome='passed'>
[35mDEBUG   [0m __name__:plugin.py:86 record_test_report:<TestReport 'test_all_outcomes.py::test_pass' when='call' outcome='passed'>
[35mDEBUG   [0m __name__:plugin.py:86 record_test_report:<TestReport 'test_all_outcomes.py::test_pass' when='teardown' outcome='passed'>
[35mDEBUG   [0m __name__:plugin.py:86 record_test_report:<TestReport 'test_all_outcomes.py::test_fail' when='setup' outcome='passed'>
[35mDEBUG   [0m __name__:plugin.py:86 record_test_report:<TestReport 'test_all_outcomes.py::test_fail' when='call' outcome='failed'>
[35mDEBUG   [0m __name__:plugin.py:86 record_test_report:<TestReport 'test_all_outcomes.py::test_fail' when='teardown' outcome='passed'>
[35mDEBUG   [0m __name__:plugin.py:86 record_test_report:<TestReport 'test_all_outcomes.py::test_skip' when='setup' outcome='skipped'>
[35mDEBUG   [0m __name__:plugin.py:103 skipping : ('test_all_outcomes.py', 8, 'Skipped: unconditional skip')
[35mDEBUG   [0m __name__:plugin.py:86 record_test_report:<TestReport 'test_all_outcomes.py::test_skip' when='teardown' outcome='passed'>]]></message></reason><output><![CDATA[[35mDEBUG   [0m __name__:plugin.py:86 record_test_report:<TestReport 'test_all_outcomes.py::test_pass' when='setup' outcome='passed'>
[35mDEBUG   [0m __name__:plugin.py:86 record_test_report:<TestReport 'test_all_outcomes.py::test_pass' when='call' outcome='passed'>
[35mDEBUG   [0m __name__:plugin.py:86 record_test_report:<TestReport 'test_all_outcomes.py::test_pass' when='teardown' outcome='passed'>
[35mDEBUG   [0m __name__:plugin.py:86 record_test_report:<TestReport 'test_all_outcomes.py::test_fail' when='setup' outcome='passed'>
[35mDEBUG   [0m __name__:plugin.py:86 record_test_report:<TestReport 'test_all_outcomes.py::test_fail' when='call' outcome='failed'>
[35mDEBUG   [0m __name__:plugin.py:86 record_test_report:<TestReport 'test_all_outcomes.py::test_fail' when='teardown' outcome='passed'>
[35mDEBUG   [0m __name__:plugin.py:86 record_test_report:<TestReport 'test_all_outcomes.py::test_skip' when='setup' outcome='skipped'>
[35mDEBUG   [0m __name__:plugin.py:103 skipping : ('test_all_outcomes.py', 8, 'Skipped: unconditional skip')
[35mDEBUG   [0m __name__:plugin.py:86 record_test_report:<TestReport 'test_all_outcomes.py::test_skip' when='teardown' outcome='passed'>]]></output></test-case><test-case asserts="0" classname="tests/integration/test_tests.py" duration="0.260025" end-time="2019-08-10 15:15:06.880183" fullname="tests/integration/test_tests.py::test_error_test" id="113" label="" methodname="test_error_test" name="tests/integration/test_tests.py::test_error_test" result="Passed" runstate="Runnable" seed="1" start-time="2019-08-10 15:15:06.620158"><properties><property name="python-version" value="3.6.8 (default, Jan 14 2019, 11:02:34) &#10;[GCC 8.0.1 20180414 (experimental) [trunk revision 259383]]" /></properties><environment clr-version="3.6.8 (default, Jan 14 2019, 11:02:34) &#10;[GCC 8.0.1 20180414 (experimental) [trunk revision 259383]]" culture="en_US" cwd="/home/adetomi/Desktop/pytest-nunit" framework-version="3.6.2" machine-name="x86_64" os-architecture="64bit" os-version="4.15.0-20-generic" platform="Linux" uiculture="en_US" user="" user-domain="" /><failure><message><![CDATA[]]></message><stack-trace><![CDATA[None]]></stack-trace></failure><reason><message><![CDATA[[35mDEBUG   [0m __name__:plugin.py:86 record_test_report:<TestReport 'test_error_test.py::test_error' when='setup' outcome='failed'>
[35mDEBUG   [0m __name__:plugin.py:86 record_test_report:<TestReport 'test_error_test.py::test_error' when='teardown' outcome='passed'>]]></message></reason><output><![CDATA[[35mDEBUG   [0m __name__:plugin.py:86 record_test_report:<TestReport 'test_error_test.py::test_error' when='setup' outcome='failed'>
[35mDEBUG   [0m __name__:plugin.py:86 record_test_report:<TestReport 'test_error_test.py::test_error' when='teardown' outcome='passed'>]]></output></test-case><environment clr-version="3.6.8 (default, Jan 14 2019, 11:02:34) &#10;[GCC 8.0.1 20180414 (experimental) [trunk revision 259383]]" culture="en_US" cwd="/home/adetomi/Desktop/pytest-nunit" framework-version="3.6.2" machine-name="x86_64" os-architecture="64bit" os-version="4.15.0-20-generic" platform="Linux" uiculture="en_US" user="" user-domain="" /></test-suite></test-run>